Science/src/temp_humidity/temp_humidity.h

```diff
@@ -1,16 +1,21 @@
-#pragma once
+#ifndef  TEMP_HUMIDITY_H
+#define  TEMP_HUMIDITY_H
 
 #include <Arduino.h>
-#include "dfrobot/DFRobot_SHT3x.h"
+#include "Wire.h"
```

Comment:
Not important for this PR, but in general, you can have two "categories" of includes: 
- libraries that contains types you are using in your API (include in your .h file)
- libraries that contain code you'll need to implement your logic (include in your cpp file)

Basically, think of it as "do others who use this file also need this library?" If so, the include goes into the header file. If not, then it can just be in the ,cpp file. So in this case, you can move the include to the .cpp file. 

Again, it doesn't actually break or change any functionality, but it's good practice as it means that including one header gives you the least amount of transitive dependencies.